packages/lib/src/toolbar/controls/InteractionHelp.tsx

```diff
@@ -0,0 +1,61 @@
+import { useClickOutside, useToggle } from '@react-hookz/web';
+import { useRef } from 'react';
+import { FiHelpCircle } from 'react-icons/fi';
+
+import styles from './InteractionHelp.module.css';
+import ToggleBtn from './ToggleBtn';
+
+const BASE_INTERACTIONS = {
+  Drag: 'Pan',
+  'Ctrl+Drag': 'Select to zoom',
+  Wheel: 'Zoom',
+};
+
+const AXIAL_ZOOM_INTERACTIONS = {
+  'Alt+Wheel': 'Zoom in X',
+  'Shift+Wheel': 'Zoom in Y',
+};
+
+interface Props {
+  hasAxialZoom?: boolean;
+}
+
+function InteractionHelp(props: Props) {
+  const { hasAxialZoom } = props;
+  const interactions = {
+    ...BASE_INTERACTIONS,
+    ...(hasAxialZoom ? AXIAL_ZOOM_INTERACTIONS : {}),
+  };
+
+  const [isHelpMenuOpen, toggleHelpMenu] = useToggle(false);
+  const rootRef = useRef(null);
+
+  useClickOutside(rootRef, () => {
+    if (isHelpMenuOpen) {
+      toggleHelpMenu(false);
+    }
+  });
+
+  return (
+    <div className={styles.root} ref={rootRef}>
+      <ToggleBtn
+        icon={FiHelpCircle}
+        iconOnly
+        onToggle={() => toggleHelpMenu()}
```

Comment:
`onToggle` doesn't receive an argument, so it's safe to just write `onToggle={toggleHelpMenu}`